### DISCUSSION TITLE
Failed to run Yarn --frozen-lockfile

### DISCUSSION THREAD

**Discussion by SlavchoSpasenoski** [getredash/redash#5981] [Q&A]
I am trying to run redash using this development guide : https://redash.io/help/open-source/dev-guide/setup

But I have issue with yarn --frozen-lockfile command.

I have had install all dependencies, requirements but this happens on Redash v10.0.0 and Redash v10.1.0, only works with Master-repo (v11.0.0-dev).

Tried with node version 12.0.0 and 14.17.2

Environment: Ubuntu 20.04

Python 3.8.10

Here is screenshot from error:

![issue](https://user-images.githubusercontent.com/28984924/231009161-e413444e-c182-4a43-9c7e-c67328a99f0c.png)

Is there any solution for this issue?

**Comment by justinclift** [comment]
As a thought, try Node 16.x when you're using the `master` branch of Redash.  Saying that not because it'll definitely solve this problem, but more because I remember it *did* solve some stuff for me ages ago.

But I don't remember exactly what the problems I was hitting were. :roll_eyes:

**Reply by SlavchoSpasenoski** [reply]
I've tried Node 16.17.04, even node 18.16.x but with no success...

Additional, tried with various versions of npm and Yarn, but still no success.

Now I have little bit different error from previous, but still is something around yarn and npm...

Still can't understand why this work in master-repo, but cannot with v10 of Redash...

![Screenshot_1](https://user-images.githubusercontent.com/28984924/231305660-38a1b05a-7219-4faf-a9f1-e3f3b7231b5f.png)

**Comment by justinclift** [comment]
Ahhh.

> Still can't understand why this work in master-repo, but cannot with v10 of Redash...

There's not a huge list of commits between master and the v10.1.0 release of Redash.  So you could manually bisect things to figure out what commit changed the behaviour, and then investigate wtf that commit is doing. :smile:
